openshift/e2e-tests.sh

```diff
@@ -10,7 +10,7 @@ export TEST_IMAGE_TEMPLATE="${EVENTING_KAFKA_BROKER_TEST_IMAGE_TEMPLATE}"
 
 env
 
-scale_up_workers || exit 1
+#scale_up_workers || exit 1
```

Comment:
All good, just curious about this one.

Why does this fail?

Reply:
it times out https://github.com/openshift-knative/eventing-kafka-broker/pull/221#issuecomment-1103819913.

Perhaps we just don't need more workers in this repository.